websocket-ballerina/tests/03_push_and_onText.bal

```diff
@@ -17,153 +17,40 @@
 import ballerina/runtime;
 import ballerina/test;
 import ballerina/io;
-import ballerina/http;
 
 string data = "";
-string expectedMsg = "{\"name\":\"Riyafa\", \"age\":23}";
 
-public type WebSocketPerson record {|
-   string name;
-   int age;
-|};
+listener Listener l2 = checkpanic new(21003);
 
-service UpgradeService /onTextString on new Listener(21003) {
-   remote isolated function onUpgrade(http:Caller caller, http:Request req) returns Service|WebSocketError {
+service /onTextString on l2 {
+   resource function onUpgrade .() returns Service|UpgradeError {
```

Comment:
Is this correct ?? Do we have an accesser named onUpgrade ?

Reply:
Changed to `get`